### ISSUE TITLE
Update contribute page

### ISSUE THREAD

**Issue by oskarth** [vacp2p/vac.dev#82]
Part of Vac.dev website update, discussed during offsite.

> Contribute/Collaborate (How to get startedddContributor’s guide, Discord, Forum, GH board, Github issues filter link (e.g. good first issue), Contribute pseudonymously, specific technical problems)

>    Daniel + Hanno
>    Richard -> Contributors “chart”
>    Giuseppe -> Contribute pseudonymously

cc @kaiserd @richard-ramos @s1fr0

**Comment by kaiserd** [comment]
I would integrate (parts of) #72 into the new `open research problems` page.
As far as I remember, I was not part of the `contribution` page effort.

**Comment by oskarth** [comment]
Right, the point is we want a contribution page, and there are many audiences for this. Some are academic, then there are others. This is what we talked about during offsite, exactly how this is divided up into one or several PRs is up to people mentioned above (from offsite notes).

Problems are a bit specific technical problems (answers "WHAT"), the Contribute page is about the social/process side of things (answers "HOW"). These are separate in purpose.

**Comment by richard-ramos** [comment]
As part of the effort to obtain the contributors chart, I created this repo: https://github.com/richard-ramos/contributors . We need to create a template to display the contributors as well as figure out how to run the script as part of the publish procedure
